src/v0/destinations/marketo_static_list/transform.js

```diff
@@ -71,76 +72,94 @@ const processEvent = (input) => {
     toRemove = getIds(message.properties.listData.remove);
   }
   if (
```

Comment:
you write this for better clarity and less conditionals:
if(condition for remove) {
// do logic for remove
}
 if(condition for add) {
// do logic for add
}

if (response.length === 0) {
throw error
}